start.sh

```diff
@@ -219,7 +202,14 @@ setupCouchbase() {
     N1QLAPI=$(getIpPort couchbase 8093)
     echo
     echo 'Creating Couchbase buckets'
-    removeBucket benchmark
+    while true; do
+        echo -n '.'
+        curl -sf -u ${COUCHBASE_USER}:${COUCHBASE_PASS} \
```

Comment:
Okay, we both want to change that at some point. Let's see where/how the rest goes.